llvm/lib/Target/AMDGPU/AMDGPUGISel.td

```diff
@@ -297,7 +297,7 @@ def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_UBYTE, SIsbuffer_load_ubyte>;
 def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_SSHORT, SIsbuffer_load_short>;
 def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_USHORT, SIsbuffer_load_ushort>;
 
-def : GINodeEquiv<G_FPTRUNC_ROUND, SIfptrunc_round>;
+def : GINodeEquiv<G_INTRINSIC_FPTRUNC_ROUND, fptrunc_round>;
```

Comment:
This should also move to SelectionDAGCompat.td

Reply:
Right. Done.